tests/test_view.py

```diff
@@ -111,190 +103,231 @@ def normalize(self, vec):
         return vec / np.linalg.norm(vec)
 
 
+@unittest.skipIf(GITHUB_ACTIONS, "GUI is not available in GitHub Actions")
+class ViewCommonCameraTC(ViewCameraTB, unittest.TestCase):
+    camera_type = "fps"  # no difference when use orbit camera
```

Comment:
I believe this comment suggests they behave the same in theory.  If that is true, both fps and orbit cameras should be tested.  Testing is required to make sure the theory is always the reality.

You may use the TB/TC pattern to test both.